package.json

```diff
@@ -110,7 +110,7 @@
     "react": "^16.8.0 || ^17.0.0 || ^18.0.0"
   },
   "engines": {
-    "node": ">=16.9.1",
+    "node": ">=18",
```

Comment:
Hey @tizmagik, typically dropping support for older node versions is considered a breaking change, but this got pulled in on a minor version.
It also looks like this is dropping support for node 16 as well, which isn't in the release notes (it only mentions node 12)

as a result, we're seeing this on Node 16.19.1 now

```
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'react-tracking@9.3.0',
npm WARN EBADENGINE   required: { node: '>=18', npm: '>=8.0.0' },
npm WARN EBADENGINE   current: { node: 'v16.19.1', npm: '8.19.3' }
npm WARN EBADENGINE }
```

Reply:
Oh sorry about that. This should have been `>=16.9`.  I can try and get a PR and re-release when I get some time, unless you have some bandwidth and would like to contribute 🙏

Reply:
I should have some time tomorrow to knock this out! thanks for the quick response

Reply:
PR is up! #215